src/site/stages/build/process-cms-exports/tests/filters.unit.spec.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable camelcase */
+
+const { expect } = require('chai');
+const { getFilter, getFilteredEntity } = require('../filters');
+
+describe('CMS export filter helpers', () => {
```

Comment:
I realized I didn't write any tests when I wrote the functionality for this. :grimacing: 

This is an attempt to remedy that.